### ISSUE TITLE
How to implement naive batch gradient descent ?

### ISSUE THREAD

**Issue by GeorgieJi** [chainer/chainer#3120]
Hi, everyone. I have a question about the implement of gradient descent. I have found several optimizers, like ada_grad, adam, sgd and so on, they're perfect. But I'm attempting to implement the naive gradient method batch gradient descent that is with the fix learning rate and acts on the whole examples in each batch. How to do it? Wait for your help. Thanks very much.

**Comment by niboshi** [comment]
Thank you for posting a question!
However, GitHub issues are for bug reports / feature requests.
Please consider asking in [StackOverflow](https://stackoverflow.com/) (with `chainer` tag).
We actively monitor it, and many others can help, too.

**Comment by GeorgieJi** [comment]
Tks. I've posted the question on the StackOverflow, wait for your help.

**Comment by niboshi** [comment]
Thank you!
